tox.ini

```diff
@@ -9,8 +9,8 @@ commands = /bin/bash -c '$(which black) --config black.ini --check .'
            /bin/bash -c '$(which mypy) --config-file mypy.ini --namespace-packages --explicit-package-bases dbt tests'
 passenv = DBT_* PYTEST_ADDOPTS
 deps =
-     -r{toxinidir}/requirements.txt
      -r{toxinidir}/dev_requirements.txt
+     -r{toxinidir}/requirements.txt
```

Comment:
This is needed because we need to install the latest dbt-spark version first?

Reply:
Yes.